django_rq/utils.py

```diff
@@ -116,12 +118,26 @@ def get_scheduler_statistics():
     return {'schedulers': schedulers}
 
 
-def get_jobs(queue, job_ids, registry=None):
+def get_jobs(
+    queue,
+    job_ids,
+    registry: Optional[
+        Union[
+            DeferredJobRegistry,
+            FailedJobRegistry,
+            FinishedJobRegistry,
+            ScheduledJobRegistry,
+            StartedJobRegistry,
+        ]
+    ] = None,
+) -> List[Job]:
     """Fetch jobs in bulk from Redis.
     1. If job data is not present in Redis, discard the result
     2. If `registry` argument is supplied, delete empty jobs from registry
     """
-    jobs = Job.fetch_many(job_ids, connection=queue.connection, serializer=queue.serializer)
+    jobs = cast(
+        List[Optional[Job]], Job.fetch_many(job_ids, connection=queue.connection, serializer=queue.serializer)
```

Comment:
this is a bug in `fetch_many` and mypy will complain that `job is None` can't happen otherwise